modules/oci-ocne-autoscaling/output.tf

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2023 Oracle Corporation and/or affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+output "instances" {
+  description = "Private IP addresses of instances."
+  value       = data.oci_core_instance.instances.*.private_ip
+}
+
+output "load_balancer_ip" {
```

Comment:
as above, please give meaningful names. outputs are like apis, so they should be clear